dev_docs/tutorials/performance/running_performance_journey_in_cloud.mdx

```diff
@@ -0,0 +1,93 @@
+---
+id: kibDevTutorialRunningPerformanceJourneyInCloud
+slug: /kibana-dev-docs/tutorial/performance/running_performance_journey_in_cloud
+title: Running Performance Journey In Cloud
+summary: Learn how to run performance journey against Cloud cluster
+date: 2023-05-04
+tags: ['kibana', 'onboarding', 'setup', 'performance', 'development', 'telemetry']
+---
+
+## Overview
+As a way to better understand user experience with Kibana in cloud, we support running performance journeys against
+Cloud deployments.
+The process takes a few steps:
+- Create a cloud deployment
+- Re-configure deployment with APM enabled and reporting metrics to the monitoring cluster
+- Create a user with `superuser` role to run tests with
+- Checkout the branch that matches your cloud deployment version
+- Run the performance journey
+
+### Re-configure deployment for Kibana and Elasticsearch
+We use [kibana-ops-e2e-perf](https://kibana-ops-e2e-perf.kb.us-central1.gcp.cloud.es.io/) cluster to monitor performance testing.
+
+If you would like to report APM metrics to this cluster, copy `SECRET_TOKEN` and `SERVER_URL` values from [packages/kbn-journeys/journey/journey_apm_config.ts](https://github.com/elastic/kibana/blob/60c82765779419d356a131e212682b69b035804b/packages/kbn-journeys/journey/journey_apm_config.ts#L10-L11)
+
+#### Change Elasticsearch configuration
+In the ESS Admin Console, find your deployment and navigate to `Security` page. Click `Add Settings` under `Elasticsearch keystore` and add new entry:
+
+```
+Setting name: tracing.apm.secret_token
+Secret: <SECRET_TOKEN>
+```
+
+Navigate to `Advanced Edit` page and change `Deployment Configuration` by adding the following JSON object to `resources.elasticsearch.plan.elasticsearch`:
+
+```
+"user_settings_override_json": {
+  "tracing.apm.enabled": "true",
+  "tracing.apm.environment": "development",
+  "tracing.apm.agent.service_name": "elasticsearch",
+  "tracing.apm.agent.server_url": "<SERVER_URL>",
+  "tracing.apm.agent.metrics_interval": "120s",
+  "tracing.apm.agent.transaction_sample_rate": "1"
+}
+```
+
+Save changes and make sure cluster is restarted successfully.
+
+#### Change Kibana configuration
+Navigate to `Advanced Edit` page and change `Deployment Configuration` by adding the following JSON object to `resources.kibana.plan.kibana`:
```

Comment:
"Advanced Edit" link wont be visible until "Edit" button is clicked.  we could start the sentence like:
 
Navigate to `Edit`-> `Advanced Edit` page ...